lib/src/store.dart

```diff
@@ -37,17 +38,21 @@ typedef StoreHandler(Store event);
 /// In a typical application using `w_flux`, a [FluxComponent] listens to
 /// `Store`s, triggering re-rendering of the UI elements based on the updated
 /// `Store` data.
-class Store {
+class Store extends Disposable {
   /// Stream controller for [_stream]. Used by [trigger].
-  StreamController<Store> _streamController;
+  final StreamController<Store> _streamController;
 
   /// Broadcast stream of "data updated" events. Listened to in [listen].
   Stream<Store> _stream;
```

Comment:
You marked the controller final, which is money, maybe we should convert this to a private getter to effectively make the stream "final" as well.